proto/proto/thin_wallet.proto

```diff
@@ -10,48 +10,13 @@ import "stake.proto";
 // Unlike the "light wallet" service, this protocol does not attempt to be
 // trust-minimized, either in terms of integrity or privacy.
 service ThinWallet {
-  rpc TransactionByNote(TransactionByNoteRequest) returns (TransactionDetail);
-  rpc AssetLookup(AssetLookupRequest) returns (chain.AssetInfo);
-  rpc AssetList(AssetListRequest) returns (stream Asset);
-  // TODO: return ValidatorStatus?
+  rpc TransactionByNote(crypto.NoteCommitment) returns (chain.NoteSource);
```

Comment:
We never used `AssetLookup`, so I deleted it.  The `AssetList` doesn't request any specific user data, so I moved it to the "light" protocol.